build_path.go

```diff
@@ -14,32 +14,28 @@ const KeyOpenAPITags = "openapi.tags"
 func buildPaths(ws *restful.WebService) spec.Paths {
 	p := spec.Paths{Paths: map[string]spec.PathItem{}}
 	for _, each := range ws.Routes() {
-		p.Paths[each.Path] = buildPathItem(ws, each)
-	}
-	return p
-}
-
-func buildPathItem(ws *restful.WebService, r restful.Route) spec.PathItem {
```

Comment:
i prefer to have this buildPathItem function separately, what if we add the item as an extra item.

+ could you provide a test for this.